hedera-node/hedera-app/src/main/java/com/hedera/node/app/workflows/handle/record/SingleTransactionRecordBuilderImpl.java

```diff
@@ -1202,4 +1202,37 @@ public EthereumTransactionRecordBuilder feeChargedToPayer(@NonNull long amount)
     public List<AccountAmount> getPaidStakingRewards() {
         return paidStakingRewards;
     }
+
+    @Override
+    public String toString() {
```

Comment:
Is there any risk of this being called in production? What is the use for this toString() method? We should be very careful with introducing toString methods, because we have found real performance problems if these get invoked (sometimes generating excessive logs). The problem is usually that someone adds a simple `"" + someObject` type line and then we find excessive time spent constructing strings.

Reply:
yes, will delete this.